NEWS

```diff
@@ -1,5 +1,30 @@
 This file lists noteworthy changes in SaunaFS.
 
+ * SaunaFS (4.6.0) (2024-10-23)
```

Comment:
I share some commits for your consideration to be included it in the `NEWS` and `changelog`:
- feat: Improve big sessions create/open speed
- ci: Add Github workflow to run Ganesha suite
- ci: Update Github workflow to run unit tests
- ci: Add new jenkins pipeline

Reply:
I left out ci specifically because it's not relevant to end users, only keeping stuff like fix, perf and feat.

As for the missing feat, interesting that it's not there, I will add it.